langkit/documentation.py

```diff
@@ -791,14 +793,19 @@ def instantiate_templates(doc_dict):
         If Handle is the root of an analysis unit, untie it and set New_Node as
         its new root. Otherwise, replace Handle with New_Node in Handle's
         parent node.
+        Note that:
+         - Handle must be tied to an existing analysis unit handle.
```

Comment:
Comments are interpreted as Sphinx, so please remove indentation for lists. Also for consistency, I would recommend to replace dashes (`-`) with stars (`*`). Thanks!